006_team_4_agent_design/006_team_4_agent_design.tex

```diff
@@ -1 +1,311 @@
+    \hline
+    Food Scarcity (food per agent) & Deaths & Days To Stabilise  \\
+        \hline
+        \hline
+        Total Scarcity ($<$3) & $>$402 & does not stabilise \\
+        \hline
+        Satisfice (=3) & 27 & 167 \\
+        \hline
+        Total Abundance ($>$5) & 0 & 0 \\
+        \hline
+    \end{tabular}
+    \caption{Table showing stabilisation time for various levels of food scarcity, reshuffling every 25 days, 500 days in total.}
+    \label{tab:scarcityStabilisationTimes}
+\end{table}
+
+\subsubsection{Effect of Random and Selfish Agents}
+
+A random agent was defined as an agent that eats random amounts of food on any given day. A selfish agent was defined as an agent that eats the amount of food it needs to remain at full HP. It was interesting to note the effect of each agent on homogeneous populations of Team 4 agents, and how many of them it took to destabilise the system.
+
+\Cref{fig:selfishagentcomps} and \Cref{fig:randomagentcomps} show the behaviour of Team 4 agents with selfish and random agents respectively. In both instances, a similar trend was noticed; there were periods of stability but the system was disrupted to an unstable state soon after. Instances with the random agent were explained by the fact that it could not communicate with other agents and therefore no meaningful relationships could be created. Hence, even if stability was achieved through the communication between Team 4 agents, the unpredictable behaviour of the random agent quickly disrupted this. 
+\newline
+With the selfish agent, despite communication being present, Team 4 agents became selfish themselves due to distrust and a lack of communication from selfish parties. This meant both agent types acted in their own self interests causing the system to destabilise and the number of deaths increase.
+
+\begin{figure}[htb]%
+    \centering
+    \subfloat[\centering 8 Team 4 agents with one selfish agent ]{{\includegraphics[width=0.45\linewidth]{006_team_4_agent_design/assets/selfishRuns/oneOtherSelfish.png}}}%
+    \qquad
+    \subfloat[\centering 8 Team 4 agents with two selfish agents ]{{\includegraphics[width=0.45\linewidth]{006_team_4_agent_design/assets/selfishRuns/twoOtherSelfish.png}}}%
+    \caption{Impact of selfish agents on system stability}%
+    \label{fig:selfishagentcomps}%
+\end{figure}
+
+\begin{figure}[htb]%
+    \centering
+    \subfloat[\centering 8 Team 4 agents with one random agent ]{{\includegraphics[width=0.45\linewidth]{006_team_4_agent_design/assets/randomRuns/oneOtherRandom.png}}}%
+    \qquad
+    \subfloat[\centering 8 Team 4 agents with two random agents ]{{\includegraphics[width=0.45\linewidth]{006_team_4_agent_design/assets/randomRuns/twoOtherRandom.png}}}%
+    \caption{Impact of random agents on system stability.}%
+    \label{fig:randomagentcomps}%
+\end{figure}
+
+Overall, it can be said that the culture of our evolutionary agent was very fragile and was easily effected by other more disruptive cultures.
```

Comment:
is 'culture' more appropriate for this section or 'behaviour'?

Reply:
It might be clearer to say strategy instead of culture, will amend this